packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -435,23 +434,23 @@ class ServiceConnectionManager {
   /// Call to returns JSON payload 'EstimateRasterCacheMemory' with two entries:
   ///   layerBytes - layer raster cache entries in bytes
   ///   pictureBytes - picture raster cache entries in bytes
-  Future<Response> get rasterCacheMetrics async {
-    if (connectedApp == null || !await connectedApp.isFlutterApp) {
+  Future<Response?> get rasterCacheMetrics async {
+    if (connectedApp == null || !await connectedApp!.isFlutterApp) {
       return null;
     }
 
     final viewId = await flutterViewId;
 
     return await _callServiceExtensionOnMainIsolate(
       registrations.flutterEngineEstimateRasterCache,
-      args: <String, String>{
+      args: <String, String?>{
```

Comment:
non-nullable. optionally, remove the types completely and let them be inferred

Reply:
done